versions/3.2.0.md

```diff
@@ -752,6 +752,7 @@ Field Name | Type | Description
 <a name="pathItemTrace"></a>trace | [Operation Object](#operationObject) | A definition of a TRACE operation on this path.
 <a name="pathItemServers"></a>servers | [[Server Object](#serverObject)] | An alternative `server` array to service all operations in this path.
 <a name="pathItemParameters"></a>parameters | [[Parameter Object](#parameterObject) \| [Reference Object](#referenceObject)] | A list of parameters that are applicable for all the operations described under this path. These parameters can be overridden at the operation level, but cannot be removed there. The list MUST NOT include duplicated parameters. A unique parameter is defined by a combination of a [name](#parameterName) and [location](#parameterIn). The list can use the [Reference Object](#referenceObject) to link to parameters that are defined at the [OpenAPI Object's components/parameters](#componentsParameters). 
+<a name="pathItemUndefinedQueryParameters"></a>undefinedQueryParameters | `boolean` | Sets the default behavior of the same field at the [Operation Object](#operationUndefinedQueryParameters) level; see that field for the description of its effects.  Default value is `true`.
```

Comment:
Should this rather be `allowUndefinedQueryParameters` in line with `allowEmptyValue` and `allowReserved`?

Reply:
@ralfhandl it would have to have a default of `true` in that case to retain minor release compatibility, but I'd be fine with making the naming more consistent.

Reply:
I don't even know what I was thinking with my last comment, the default is `true` with the current name already.  I'll make the name change, although this is still blocked on the discussion in the issue.